### PR TITLE
pt: fix loss training when no data available

### LINKED ISSUE #3482: [BUG] pt: virial training seems unsupported
### Bug summary

No virial loss is outputted even when `start_pref_v` and `limit_pref_v` are changed to non-zero values.

### DeePMD-kit Version

v3.0.0a0-28-ged831c88-dirty

### TensorFlow Version

PT v2.2.0+cu121-g8ac9b20d4b0

### How did you download the software?

From source

### Input Files, Running Commands, Error Log, etc.

`lcurve.out` doesn't show anything for virial.

```
#  step      rmse_val    rmse_trn    rmse_e_val  rmse_e_trn    rmse_f_val  rmse_f_trn         lr
      0      2.76e+01    2.61e+01      5.67e-01    9.28e-01      8.72e-01    8.25e-01    1.0e-03
    100      9.46e+00    9.62e+00      1.87e-01    2.06e-01      2.99e-01    3.04e-01    1.0e-03
```

### Steps to Reproduce

`start_pref_v` and `limit_pref_v` are changed to 
```diff
diff --git a/examples/water/se_e2_a/input_torch.json b/examples/water/se_e2_a/input_torch.json
index fe424afe..b41cd35b 100644
--- a/examples/water/se_e2_a/input_torch.json
+++ b/examples/water/se_e2_a/input_torch.json
@@ -49,6 +49,8 @@
     "limit_pref_e": 1,
     "start_pref_f": 1000,
     "limit_pref_f": 1,
+    "start_pref_v": 1,
+    "limit_pref_v": 1,
     "_comment": " that's all"
   },
   "training": {
```

Then run
```sh
cd examples/water/se_e2_a
dp --pt train input_torch.json
```

### Further Information, Files, and Links

_No response_

### PR DESCRIPTION
**Pull request by iProzd**
Fix #3482 and #3483.

### CODE REVIEW

**Comment by njzjz** [comment]
I understand it could fix #3483, but what was the problem with #3482?

**Review by njzjz — APPROVED** [review]
See @wanghan-iapcm's comment in #3047

**Comment by iProzd** [comment]
> I understand it could fix #3483, but what was the problem with #3482?

Strangely, I can not reproduce #3482 even on current devel branch.

**Comment by iProzd** [comment]
> See @wanghan-iapcm's comment in #3047

Fixed in https://github.com/deepmodeling/deepmd-kit/pull/3571/commits/d5e9f887c1d33f6b2a752b473e861862546ef331

**Comment by njzjz** [comment]
> > I understand it could fix #3483, but what was the problem with #3482?
> 
> Strangely, I can not reproduce #3482 even on current devel branch.

I rechecked it and I think you are right. I may made a mistake here... I'll close #3482 directly.

**Review by wanghan-iapcm — COMMENTED** [review]
How do you test if the loss work property when the label is absent?

**Review by wanghan-iapcm — APPROVED** [review]
